_posts/2017/11/2017-11-08-online-meetings.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+authors: ["Belinda Weaver"]
+title: "Running effective online meetings with Zoom (or Google Hangouts)"
+date: 2017-11-08
+time: "00:00:00"
+category: ["Online meetings", "Zoom"]
+---
+ 
+Online meetings are a fact of life for most of us in the Carpentries. So how do we make them as effective as we can?
+
+This post came about after the Twitter discussion generated by this [tweet from Titus Brown](https://twitter.com/ctitusbrown/status/926165114948001792).
+
+Here are some points I have thought of. Feel free to comment.
+
+#### Before the meeting:
+
+1. Use a headset. You will hear and be heard much better. The headphones from a smart phone are sufficient. Aything is better than using the in-built microphone from your computer as it will pick up a lot of other noise.
+
+2. Prepare for the meeting by testing your set up well before the meeting starts. Can you hear? Be heard? Do you know how to mute yourself? This is important as people can make a lot of noise if they type during the meeting.
+
+3. Mute your cell phone in case it rings or pings during the meeting.
+
+4. Move to a quiet place if you can so outside noise does not intrude on others.
+
+5. Please don't be the person who comes in late, without a headset, and makes a lot of noise from screechy feedback!
+
+#### In the meeting:
+
+The Chair sets the tone. The Chair introduces him or herself, explains how the meeting will be conducted, and what expectations there are of attendees. The Chair then calls on any notetaker for the meeting to introduce him or herself.
+
+The chair greets each person as that person joins the meeting. Once the meeting starts, the Chair calls on each person in turn to say their name, where they are from, and why they are attending, e.g.instructor checkout, workshop debrief.
+
+The Chair asks people who wish to speak to write the word `hand` in the chat window. The Chair then calls on speakers in turn. 
+
+Notes are recorded on an etherpad. Any notes posted to the chat window during the meeting should be transferred to the etherpad so they are not lost when the zoom is closed.
+
+#### Three rules to make for better meetings
+
+- Be on time - it is very disruptive when people join an online meeting after it has started.
```

Comment:
Be prepared for the meeting - meetings where people know what's going on are more effective and will take less time.